### ISSUE TITLE
Llama examples not working.

### ISSUE THREAD

**Issue by Anindyadeep** [tinygrad/tinygrad#2825]
After cloning the repo, when I ran:

```bash
python3 examples/llama.py --model ../../models/llama-2-7b-raw/
```

It gives this error:

```
Traceback (most recent call last):
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/examples/llama.py", line 15, in <module>
    from extra.models.llama import Transformer, convert_from_huggingface
ModuleNotFoundError: No module named 'extra'
```

For a quick fix, I have also put `tinygrad/extras` to `tinygrad/tinygrad/extras` and then ran the same command once again, and I got this error:

```
using CUDA backend
using LLaMA-7B model
ram used:  0.00 GB, layers.0.attention.wq.weight                      :   0%|                                                                 | 0/292 [00:00<?, ?it/s]
loaded weights in 28.96 ms, 0.03 GB loaded at 1.16 GB/s
Traceback (most recent call last):
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/examples/llama.py", line 373, in <module>
    llama = LLaMa.build(MODEL_PATH, TOKENIZER_PATH, model_gen=args.gen, model_size=args.size, quantize=args.quantize)
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/examples/llama.py", line 174, in build
    load_state_dict(model, weights, strict=False)
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/nn/state.py", line 68, in load_state_dict
    v.assign(state_dict[k].to(v.device)).realize()
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/tensor.py", line 107, in realize
    run_schedule(self.lazydata.schedule())
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/realize.py", line 28, in run_schedule
    prg = lower_schedule_item(si)
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/realize.py", line 19, in lower_schedule_item
    return Device[si.out.device].get_runner(si.ast)
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/device.py", line 314, in get_runner
    def get_runner(self, ast:LazyOp) -> CompiledASTRunner: return self.to_program(self.get_linearizer(ast))
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/device.py", line 286, in to_program
    return CompiledASTRunner(k.ast, k.name, src, k.global_size, k.local_size, runtime_args).build(self.compiler, self.runtime)
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/device.py", line 254, in build
    self.lib = compiler.__wrapped__(self.prg) if getenv("DISABLE_COMPILER_CACHE") else compiler(self.prg)
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/helpers.py", line 262, in wrapper
    return diskcache_put(table, key, func(*args, **kwargs))
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/runtime/ops_cuda.py", line 23, in compile_cuda
    def compile_cuda(prg) -> bytes: return compile_cuda_style(prg, [f'--gpu-architecture={CUDADevice.default_arch_name}', "-I/usr/local/cuda/include", "-I/usr/include"], cuda.nvrtcProgram, cuda.nvrtcCreateProgram, cuda.nvrtcCompileProgram, cuda.nvrtcGetPTX, cuda.nvrtcGetPTXSize, cuda.nvrtcGetProgramLog, cuda.nvrtcGetProgramLogSize, check)  # noqa: E501
  File "/home/paperspace/workspace/benchmarks/bench_tinygrad/tinygrad/tinygrad/helpers.py", line 323, in compile_cuda_style
    if status != 0: raise RuntimeError(f"compile failed: {get_bytes(prog, get_log_size, get_log, check).decode()}")
RuntimeError: compile failed: <null>(3): error: identifier "__bf16" is undefined
  extern "C" __global__ void E_131072_32_4(unsigned int* data0, const __bf16* data1) {
                                                                      ^

<null>(6): error: identifier "__bf16" is undefined
    __bf16 val0 = data1[(gidx0*128)+(lidx1*4)];
    ^

<null>(7): error: identifier "__bf16" is undefined
    __bf16 val1 = data1[(gidx0*128)+(lidx1*4)+1];
    ^

<null>(8): error: identifier "__bf16" is undefined
    __bf16 val2 = data1[(gidx0*128)+(lidx1*4)+2];
    ^

<null>(9): error: identifier "__bf16" is undefined
    __bf16 val3 = data1[(gidx0*128)+(lidx1*4)+3];
    ^

5 errors detected in the compilation of "<null>".
```

**Comment by beverm2391** [comment]
I've had similar issues importing that module. I wasn't able to reproduce this, but I'd be interested to see the output of `extra.__path__` and `sys.path`

**Comment by g1y5x3** [comment]
it should be fixed with #3036
